packages/project-utils/bundling/function/createWatchConfig.js

```diff
@@ -8,25 +8,29 @@ module.exports = options => {
     const output = getOutput(options);
     const entry = getEntry(options);
 
-    const { overrides, debug, cwd } = options;
+    const { overrides, cwd } = options;
 
     // Customize Babel options.
     if (typeof overrides.babel === "function") {
         babelOptions = overrides.babel(babelOptions);
     }
 
+    const sourceMaps = process.env.SOURCE_MAPS !== "false";
```

Comment:
You know what... when I think about it, let's not even check an environment variable, and have this hardcoded to `true`.

The reason being.... users can always adjust the Webpack configs on their own (via their `webiny.config.ts` files). They have the ability to turn off this feature there.

Ultimately, it's probably better to leverage existing features we already have in place. This way we also don't have to write any documentation that cover this magic env variable.

Reply:
It's used in several places, so user would have to override all of them to effectively turn it off. I'll better move it to `options` object.

Reply:
Yes, but one thing to note here - it's possible that the number of `webiny.config.ts` and workspaces will reduce in the future (been experimenting with this in our WCP repo), so ultimately, users would only need to do a single adjustment, not 10+ which is now the case.

Just something to have in mind.